### PR TITLE
[tweak] - Switch linode labels to LinodeMachine Name instead of UID

### PR DESCRIPTION
**Pull request by komer3**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:
This PR switched to using linode machine names instead of UID for less confusion

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**TODOs**:

- [x] squashed commits
- [ ] includes documentation
- [ ] adds unit tests
- [ ] adds or updates e2e tests

### CODE REVIEW

File: controller/linodevpc_controller_helpers.go [hunk 1]
```diff
@@ -41,7 +41,7 @@ func (r *LinodeVPCReconciler) reconcileVPC(ctx context.Context, vpcScope *scope.
 	}
 
 	if createConfig.Label == "" {
-		createConfig.Label = util.RenderObjectLabel(vpcScope.LinodeVPC.UID)
```
Comment (eljohnson92) [inline controller/linodevpc_controller_helpers.go:44]: are there other usages of renderObjectLabel? or can this function be deleted?

Comment (AshleyDumaine) [reply controller/linodevpc_controller_helpers.go:44]: Yeah, my IDE shows no other usages so we should be safe to remove this

**Comment by AshleyDumaine** [comment]
Aside from cleaning up the `RenderObjectLabel` function, LGTM 👍

File: controller/linodevpc_controller_helpers.go (hunk 1 above)
Comment (komer3) [reply controller/linodevpc_controller_helpers.go:44]: Deleted the function. It doesn't affect anything else

**Review by AshleyDumaine — APPROVED** [review]

**Review by eljohnson92 — APPROVED** [review]

**Pull request merged**